src/test/models/users.model.js

```diff
@@ -6,29 +6,20 @@ import '../../server/models/users.model';
  * Test for mongodb create.
  */
 
-let should = require('should');
+const should = require('should');
 let user = require('mongoose').model('user');
+let u = {
+  name: 'yoda'
+};
 
 describe('Users: models', function () {
   describe('#create()', function () {
-    it('should create a new User', function (done) {
+    it('should create a new User and remove', function (done) {
       // Create a User object to pass to User.create()
-      var u = {
-        name: 'yoda'
-      };
       user.create(u, function (err, createdUser) {
         should.not.exist(err);
         createdUser.name.should.equal('yoda');
-        done();
-      });
-    });
-  
-    it('should delete new User', function (done) {
-      // Create a User object to pass to User.create()
-      var u = {
-        name: 'yoda'
-      };
-      user.find(u).remove(function() {
+        createdUser.remove();
```

Comment:
I see. Then at the beginning of every test case, we need a routine to clear the entire DB. One of the URLs I sent about Mocha was doing that by setting up an environment function that gets run before every unit test. For now, I thought we should just do that manually.